lua/wire/server/wirelib.lua

```diff
@@ -1058,10 +1058,21 @@ function WireLib.dummytrace(ent)
 	}
 end
 
+function WireLib.NumModelSkins(model)
+	if NumModelSkins then
```

Comment:
Why the if?
Are there cases where it does exist?
If so, who creates it?
Can we depend on them implementing it correctly now and for all eternity?